admin/load.php

```diff
@@ -126,16 +126,18 @@ function perflab_render_modules_page() {
  * @param array  $module_settings Associative array of the module's current settings.
  */
 function perflab_render_modules_page_field( $module_slug, $module_data, $module_settings ) {
-	$base_id   = sprintf( 'module_%s', $module_slug );
-	$base_name = sprintf( '%1$s[%2$s]', PERFLAB_MODULES_SETTING, $module_slug );
-	$enabled   = isset( $module_settings['enabled'] ) && $module_settings['enabled'];
+	$base_id          = sprintf( 'module_%s', $module_slug );
+	$base_name        = sprintf( '%1$s[%2$s]', PERFLAB_MODULES_SETTING, $module_slug );
+	$enabled          = isset( $module_settings['enabled'] ) && $module_settings['enabled'];
+	$plugins_config   = perflab_get_standalone_plugins_config();
+	$is_active_plugin = ( isset( $plugins_config[ $module_slug ] ) && ( '' !== $plugins_config[ $module_slug ] ) && is_plugin_active( $plugins_config[ $module_slug ] ) ) ? true : false;
 	?>
 	<fieldset>
 		<legend class="screen-reader-text">
 			<?php echo esc_html( $module_data['name'] ); ?>
 		</legend>
 		<label for="<?php echo esc_attr( "{$base_id}_enabled" ); ?>">
-			<?php if ( perflab_can_load_module( $module_slug ) ) { ?>
+			<?php if ( perflab_can_load_module( $module_slug ) && ! $is_active_plugin ) { ?>
```

Comment:
See my other comment: If we go with the constants approach and with modifying `perflab_can_load_module()`, we could change the logic here:
* If `perflab_can_load_module()` returns a `WP_Error` or `false`, it means the module cannot be loaded.
* If a `WP_Error` is returned, we should then display the error message from that error. And only show the default fallback messages below if `false` is returned.

Reply:
@felixarntz The constant logic did not work here because the module `load.php` file ran before this check, so it was for the single Performance Lab plugin, and it shows `The module cannot be managed with Performance Lab since it is already active as a standalone plugin.`

Reply:
@mukeshpanchal27 Evan after you did what I recommended in https://github.com/WordPress/performance/pull/670#discussion_r1137503887?

Reply:
@mukeshpanchal27 That is functioning as expected: If the WebP Uploads plugin is active, the module checkbox _should_ be disabled with the message you're saying. So this is all good as is.